lib/msal-core/package.json

```diff
@@ -45,6 +45,7 @@
     "prepack": "npm run lint && npm test && npm run build"
   },
   "dependencies": {
+    "caniuse-lite": "^1.0.30001255",
```

Comment:
Can this be removed?

Reply:
Sure thing, don't know why it was initially installed in the first place.